.github/workflows/package.yml

```diff
@@ -0,0 +1,87 @@
+name: Packages
+on: [push, pull_request]
+
+jobs:
+  build-deb:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        distro: [debian-unstable, debian-bullseye, debian-buster, ubuntu-groovy, ubuntu-focal, ubuntu-eoan]
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: legoktm/gh-action-auto-dch@master
+        with:
+          fullname: Kiwix builder
+          email: robot@kiwix.build
```

Comment:
OK. Does that email forward to anyone?

Does that address already have a GPG key? Or should I create a new one for it? That was the main reason I used a dummy email for this, so it wouldn't conflict with any other potential keys.

Reply:
> OK. Does that email forward to anyone?

Yes.

> Does that address already have a GPG key? Or should I create a new one for it? That was the main reason I used a dummy email for this, so it wouldn't conflict with any other potential keys.

Probably not. It's probably to @kelson42 do it (or see with him at least)

Reply:
A little more detail:

This email address is used in the debian/changelog. Every time the CI builds a package it generates an entry like:
```
zimlib (6.1.4~git202006020952.61208f5~bullseye) bullseye; urgency=medium

  * New automated build

 -- Kiwix builder <robot@kiwix.build>  Tue, 02 Jun 2020 09:56:20 +0000
```

Users will never see this unless they go digging for it (`/usr/share/doc/libzim-dev/changelog.gz` or something). 

The name & email combo `Kiwix builder <robot@kiwix.build>` is tied to the secret GPG key I provided to Kelson, which he configured in Github secrets. The main reason I wanted to attach the key to a non-existent email address was so that no one treated it like a "real" GPG key used to encrypt email or sign stuff, because all we're using it for is to build packages in an automated fashion.

For now I would prefer to get this working with the current fake address, and if we deem it's a problem later on, it should be relatively straightforward to change it (generate a new key, then configure it in launchpad and set it in github secrets...probably half an hour).

Reply:
Good for me, even if I don't really see the risk of having a proper email address.

Reply:
I don't see why we should not use a valid email address. (At least we should must use a valid domain name "kiwix.org", and if we change it, we can change it to a valid address).

There are several place where we are push things automatically (on kiwix-desktop flathub repository, android/ios app on stores, ...) I think we should use the same address all the time (and potentially the same gpg key, even if each "store" has probably its own authentication system). And if its somehow because the official address for release, we should use a real address (potentially reusing a another official kiwix gpg key)

Reply:
The GPG key is used solely to authenticate with launchpad and nothing else. And in my opinion, we should be using a dedicated key for that and not sharing it anywhere else. This key gives root access to users' devices.

But GPG keys are also used for signing/encrypting emails (and some other less important things). I don't want people to accidentally find a `kiwix.org` key and try using it. If we use a real email address, then the person responsible for that address should be generating and maintaining the key...but I don't know how we could ever consider that key to be secure given that GitHub also has a copy of it. What if we want to GPG sign releases of some tarball in the future?

I think it's also worth considering that no user is really ever going to see this email address, it's mostly an internal implementation detail as it's what is associated with the GPG key.

If it's really a problem, I suppose we can create some launchpad@kiwix.org email? But I don't think it's worth it.

Reply:
@legoktm Your last proposal goes in the right direction. It's OK for me but if we try to bridge your two proposals, what about `release+launchpad@kiwix.org`?

Reply:
Yes, I think `release+launchpad@kiwix.org` addresses my concerns. I'll send you a new key shortly.